### DISCUSSION TITLE
TP5 - Producto escalar (ejercicio 1, punto 2)

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#159] [Preguntas y respuestas] [answered]
Hola buenas! Que es lo que tendria que retornar la funcion del punto 2?  

```int producto_escalar(int filas, int columnas, int matriz[][], int escalar);```

**Comment by martinvilu** [comment, marked as answer]
Por lo menos, verificar que la matriz sea válida y que las dimensiones sean apropiadas.

**Reply by agusscarava** [reply]
osea retornar por ejemplo que el producto se pudo realizar correctamente, porque no se deberia modificar la dimension de la matriz. o si?

**Reply by martinvilu** [reply]
> porque no se deberia modificar la dimension de la matriz. o si?

Como está definida la función, ¿es posible? (cambiar la dimensión de la matriz)
